tests/legacy-cli/e2e/tests/build/lazy-load-syntax.ts

```diff
@@ -5,17 +5,36 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-import { readFile, replaceInFile, writeFile } from '../../utils/fs';
+import { getGlobalVariable } from '../../utils/env';
+import { appendToFile, prependToFile, readFile, replaceInFile, writeFile } from '../../utils/fs';
 import { ng } from '../../utils/process';
-import { createProject, updateJsonFile } from '../../utils/project';
+import { updateJsonFile } from '../../utils/project';
 import { expectToFail } from '../../utils/utils';
 
 export default async function () {
-  const projectName = 'ivy-lazy-loading';
+  const argv = getGlobalVariable('argv');
+  const ivyProject = argv['ivy'];
```

Comment:
How are these run during the e2e-cli tests?

Isn't the `ivy` arg used only in the cli snapshots?

Reply:
Yes, Ivy tests are only run for snapshots now. I agree with you that we would love coverage. I think moving the ivy job to not use the snapshots, and to always run, is a good approach. We're starting to have more cde that interacts with Ivy so we should have confidence in PRs for it.